src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -0,0 +1,29 @@
+package racingcar.domain
+
+import racingcar.generator.NumberGenerator
+
+class Car(
+    val name: String,
+) {
+    var position: Int = 0
+        private set
```

Comment:
I was thinking of declaring the field this way, so I’m wondering if there’s a specific reason for doing it differently.
```suggestion
class Car(
    private var position: Int,
    val name: String,
) {
```

Reply:
I thought I didn't need to include the `position` variable in the primary constructor, since it's always initialized default value of 0.
I also needed to expose `get()` externally while hiding `set()`.